tests/test_app_server.py

```
@@ -2370,6 +2370,363 @@ async def json(self):
+@pytest.mark.asyncio
+async def test_memory_browse_endpoint_falls_back_to_shared(monkeypatch, tmp_path):
+    """GET /memory/browse for a URI that lives only in __shared__ must
+    surface the shared content to the desktop client via read fallback.
+    Pins the engine.recall(fallback_to_shared=True) contract at the
+    HTTP boundary."""
+    pytest.importorskip("fastapi")
+
+    from omicsclaw.app import server
+    from omicsclaw.memory.memory_client import MemoryClient
+
+    graph, _, memory_pkg = await _setup_memory_review_runtime(monkeypatch, tmp_path)
```

Ruff (0.15.12)

[warning] Unpacked variable `graph` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)